### PR TITLE
docs: Add badges

### LINKED ISSUE #60: Add badges
We should add badges for various things:
- [ ] docs
- [ ] npm
- [ ] Ryver chat

See https://github.com/badges/shields

### PR DESCRIPTION
**Pull request by abmusse**
Resolves #60

### CODE REVIEW

**Comment by markdirish** [comment]
Sleek. I know I'm not on the reviewers, but LGTM!

**Comment by abmusse** [comment]
> Sleek. I know I'm not on the reviewers, but LGTM!

Thanks! Added you on as reviewer.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![npm](https://img.shields.io/npm/v/idb-pconnector?logo=npm)](https://www.npmjs.com/package/idb-pconnector)
+[![ryver](https://img.shields.io/badge/Ryver-Chat-blue)](https://ibmioss.ryver.com/index.html#forums/1000127)
```
Comment (kadler) [inline README.md]: They only problem with this link is that Ryver gives no way to sign up if they don't already have an account. For that, they need the signup link: https://ibmioss.ryver.com/application/signup/members/9tJsXDG7_iSSi1Q

**Comment by abmusse** [comment]
@kadler

I have updated the link and also removed the issue number from the commit.

**Comment by kadler** [comment]
Really, we need both links. Having a link to the specific forum _and_ a signup link (since Ryver doesn't give you any way to sign up or view without signing up).

**Comment by abmusse** [comment]
> Really, we need both links. Having a link to the specific forum _and_ a signup link (since Ryver doesn't give you any way to sign up or view without signing up).

Gotcha I will add another badge to sign up.

**Comment by abmusse** [comment]
Added signup badge.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+[![npm](https://img.shields.io/npm/v/idb-pconnector?logo=npm)](https://www.npmjs.com/package/idb-pconnector)
+[![ryver-signup](https://img.shields.io/badge/Ryver-Signup-blue)](https://ibmioss.ryver.com/application/signup/members/9tJsXDG7_iSSi1Q)
+[![ryver-chat](https://img.shields.io/badge/Ryver-Chat-blue)](https://ibmioss.ryver.com/index.html#forums/1000127)
```
Comment (kadler) [inline README.md]: Can you swap these two?

Comment (abmusse) [reply README.md]: Done.

**Pull request merged**